app/index.js

```diff
@@ -75,7 +75,20 @@ module.exports = yeoman.generators.Base.extend({
       this.includeBootstrap = hasFeature('includeBootstrap');
       this.includeModernizr = hasFeature('includeModernizr');
 
-      done();
+      // give the option to exclude jquery if bootstrap isn't selected
+      if(!this.includeBootstrap) {
+        this.prompt({
+          type: 'confirm',
+          message: 'Include jQuery',
+          name: 'includeJQuery',
+          default: true
+        }, function(jQueryAnswer) {
+          this.includejQuery = jQueryAnswer.includeJQuery;
```

Comment:
Why aren't you consistent about the name? Some identifiers use capital `J`s while others use `j`.